src/main/java/com/softserve/app/controller/UserController.java

```diff
@@ -1,32 +1,78 @@
 package com.softserve.app.controller;
 
+import com.softserve.app.constant.SportHubConstant;
 import com.softserve.app.dto.UserDTO;
 import com.softserve.app.models.User;
 import com.softserve.app.service.UserService;
 import lombok.AllArgsConstructor;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.util.Objects;
 
 
 @RestController
+@CrossOrigin(origins = "http://localhost:4200")
 @AllArgsConstructor
+@RequestMapping("/user/{user_id}")
 public class UserController {
 
-
     // todo: user registration and login
 
-
     private final UserService userService;
 
-    // todo: add methods
 
+    @GetMapping
+    public ResponseEntity<Object> showProfile(
```

Comment:
return only DTO, if you want to return some status. Throw your custom exception in service with your http code